Common/FileUtil.h

```diff
@@ -26,12 +26,14 @@
 #include "Common.h"
 
 #ifdef _WIN32
```

Comment:
This should probably simply be changed to #ifdef _MSC_VER to identify MS' compiler and libs.